app/src/components/pageLayout/pageLayout.tsx

```diff
@@ -21,6 +22,22 @@ interface PageLayoutProps {
   adobeAnalyticsPageName?: string;
 }
 
+type FeedbackContextType = {
+  onOpen: () => void;
+};
+
+export const FeedbackContext = createContext<FeedbackContextType | undefined>(
+  undefined
+);
+
+export const useFeedbackContext = () => {
+  const context = useContext(FeedbackContext);
+  if (!context) {
+    throw new Error("useFeedbackContext must be used within its provider");
+  }
+  return context;
+};
```

Comment:
Create a new directory in `/app/src` for the app's `context` and this part can be moved there. Besides `FeedbackContext` and `useFeedbackContext`, you can also create a `FeedbackProvider` that contains the code for `<FeedbackContext.Provider value={{ onOpen }}>`. 

As the app grows and you have more context, it might be easier to read
```
<FeedbackProvider ...>
  <SomeOtherProvider ...>
    <SkipNavigation />
    <OtherComponents />
  </SomeOtherProvider>
</FeedbackProvider>
```

Rather than 
```
<FeedbackContext.Provider value={{ onOpen }}>
  ....
```

Try it out and let me know what you think, but some of the code inside `PageLayout` will have to be moved out.